rpc/src/v1/helpers/light_fetch.rs

```diff
@@ -87,6 +87,7 @@ pub fn extract_transaction_at_index(block: encoded::Block, index: usize, eip86_t
 }
 
 /// Type alias for convenience.
+#[allow(dead_code)]
```

Comment:
Did you figure out why the compiler thinks `ExecutionResult` is not used? I had a stab a while back but didn't dig very deep.

Reply:
@dvdplm looks like the problem was that the type was unnecessary public.
I used Marek's suggestion though.